src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public enum Colors {
+    RED,
+    ORANGE,
+    YELLOW,
+    GREEN,
+    BLUE,
+    NAVY,
+    PURPLE;
+
+    @Override
+    public String toString() {
+        return name().charAt(0) + name().substring(1).toLowerCase();
+    }
```

Comment:
you don't need to override the method toString(), just use name() or values (see example here https://howtodoinjava.com/java/enum/java-enum-string-example/)